package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "gpii-pouchdb-lucene",
+  "version": "0.1.0",
+  "description": "A wrapper for couchdb-lucene to allow it to be required and used in tests.",
+  "main": "index.js",
+  "config": {
+    "url": "https://github.com/rnewson/couchdb-lucene.git",
+    "hash": "ce773de1becf8a400e0344a43cf6eea3134dcc63",
+    "version": "1.0.2",
+    "srcDir": "src/couchdb-lucene",
+    "distDir": "dist"
+  },
+  "scripts": {
+    "test":        "node tests/all-tests.js",
+    "clone":       "rm -rf $npm_package_config_srcDir && git clone $npm_package_config_url $npm_package_config_srcDir",
+    "checkout":    "cd $npm_package_config_srcDir && git checkout $npm_package_config_hash",
+    "build":       "cd $npm_package_config_srcDir && mvn -Dmaven.test.skip=true",
+    "postinstall": "grunt dedupe-infusion && npm run clone && npm run checkout && npm run build"
+  },
+  "author": "Tony Atkins <tony@raisingthefloor.org>",
+  "license": "See LICENSE file.",
+  "dependencies": {
+    "infusion": "git://github.com/fluid-project/infusion.git#605b0bcfaedf6193c31456236d3aa4e42cf36a96",
+    "request": "2.54.0"
+  },
+  "devDependencies": {
+    "adm-zip":              "^0.4.7",
+    "gpii-express":         "git://github.com/GPII/gpii-express#d2b57d7920270926a0fad77c81a39541a2a1a6e6",
+    "gpii-pouch":           "git://github.com/the-t-in-rtf/gpii-pouchdb.git#55b8b3bafb170183985cc529cf84bf00caee5fae",
```

Comment:
Done.  In doing so I discovered GPII-1259, the fixes for which are now in flight.